### PR TITLE
add dep and cmd test

### PR DESCRIPTION
**Pull request by RNAer**

### CODE REVIEW

File: recipes/deblur/build.sh [hunk 1]
```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
-pip install biom-format
```
Comment (wasade) [inline recipes/deblur/build.sh:2]: per @mortonjt comment, this might be the source of the future issue?

Comment (mortonjt) [reply recipes/deblur/build.sh:2]: but we need to install the biom-format?

File: recipes/deblur/meta.yaml [hunk 2]
```diff
@@ -22,6 +24,8 @@ requirements:
     - pip
 
   run:
+    - biom-format
```
Comment (mortonjt) [inline recipes/deblur/meta.yaml:27]: Ahh.  I see it now

**Review by mortonjt — APPROVED** [review]

**Comment by wasade** [comment]
Should this get merged?

**Comment by wasade** [comment]
Thanks! Do you know how long it takes for these changes to propagate? I just did the install again and it looks like it's still messing up on biom

**Comment by RNAer** [comment]
depending on the queue on travis. should be finished shortly. the status
can be checked here: https://travis-ci.org/biocore/conda-recipes

**Comment by wasade** [comment]
Great, thanks!

**Pull request merged**